src/main/webapp/js/userMapData.json

```diff
@@ -571,6 +624,7 @@
       "Saint Louis University",
       "Saint Mary's University",
       "University of Manila",
+      "University of St. La Salle",
```

Comment:
University of St. La Salle and De La Salle University are different universities. University of St. La Salle is located in Bacolod, Philippines whereas De La Salle University is located in Manila, Philippines.